airflow/plugins_manager.py

```diff
@@ -183,103 +202,116 @@ def make_module(name: str, objects: List[Any]):
 # pylint: enable=protected-access
 
 
-# Plugin components to integrate as modules
-operators_modules = []
-sensors_modules = []
-hooks_modules = []
-executors_modules = []
-macros_modules = []
+def load_plugins():
```

Comment:
How about this as a name?

```suggestion
def ensure_plugins_loaded():
```

It makes it clearer from the call-site that it will only load them once, not every time this fn is called

Reply:
Done.